chainer/functions/__init__.py

```diff
@@ -22,6 +22,8 @@
 from chainer.functions.activation.prelu import prelu  # NOQA
 from chainer.functions.activation.relu import relu  # NOQA
 from chainer.functions.activation.relu import ReLU  # NOQA
+from chainer.functions.activation.rrelu import rrelu  # NOQA
+from chainer.functions.activation.rrelu import RReLU  # NOQA
```

Comment:
Please remove upper case `RReLU`. We no longer expose function class aliases.

Reply:
I modified.

Reply:
I need to instantiate the function class directly in value to reuse the mask, because f will be called repeatedly.
I refered test_dropout.py in master branch.

Reply:
Sorry, we've changed the policy.  See #4907.

Reply:
I adopted the new policy.